.semaphore/release/test-release-build.yml

```diff
@@ -5,9 +5,6 @@ agent:
     type: e1-standard-4
     os_image: ubuntu2004
 
-execution_time_limit:
```

Comment:
You can't remove this - removal means that semphore will default to 1 hour.  And then it will complain that the job timeout is longer than the pipeline timeout.

You need to leave this in, but increase the pipeline timeout to be much longer than the job timeout.

Reply:
Thanks for the catch!